lago/providers/libvirt/vm.py

```diff
@@ -661,6 +684,13 @@ def _libvirt_xml(self):
                     type='virtio',
                 ),
             )
+            interface.append(
+                ET.Element(
+                    'driver',
+                    name='vhost',
```

Comment:
I think that it would be better to drop the `name` attribute.
Libvirt's default is `vhost`, and if this requirement can be provided by the kernel, libvirt will fallback to use `qemu`.